CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 ## Unreleased
 <!-- Add new, unreleased items here. -->
 
+## v2.0.0 [10-16-2017]
+- [BREAKING] `Workspace.init()` options have been moved to the constructor.
+- [BREAKING] `Workspace` dependency installation must be called manually via
+`Workspace.installBowerDependencies()`.
+- [BREAKING] `Workspace` Class is now responsible for setup & initialization only. All
+post-setup operations (`pushChangesToGithub`, `commitChanges`, etc.) have been
+pulled off as seperate operations.
+- `run()`: New batch-running support for running commands across multiple repos.
+- Updated README.
```

Comment:
Recommend adding a .github/PULL_REQUEST_TEMPLATE with a checkbox to check for changelog updating.

This is the only way I ever remember to update any changelogs -.-

Reply:
+1

Reply:
+1000, grabbing from tools-common now